.github/workflows/ci.yaml

```diff
@@ -86,3 +86,18 @@ jobs:
       - name: Check project
         run: |
           pkgmt check
+          
+  docs-upload:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: us-east-1
+          role-to-assume: arn:aws:iam::888374473732:role/jupysql-ci-github-actions
```

Comment:
do we need this? I think that adding the pre_build and post_build tasks in the readthedocs config should be enough

Reply:
No we dont need this. I was just experimenting and forgot to remove it.